library/private/camera_impl.h

```diff
@@ -57,6 +57,7 @@ class camera_impl : public camera
   camera& yaw(angle_deg_t angle) override;
   camera& elevation(angle_deg_t angle) override;
   camera& pitch(angle_deg_t angle) override;
+  camera& ortoview() override;
```

Comment:
English is not my native language but it seems like a `h` is missing.  
Please, someone confirm?
![image](https://github.com/f3d-app/f3d/assets/6069752/adbafe49-45a4-4efe-b316-9049d31af3a9)